src/control-plane/backend/lambda/api/model/stacks.ts

```diff
@@ -1048,6 +1060,9 @@ export class CDataModelingStack extends JSONObject {
   @JSONObject.optional('')
     ClickstreamAnalyticsMetadataDdbArn?: string;
 
+  @JSONObject.optional('')
+    ClickstreamMetadataDdbArn?: string;
```

Comment:
@tyyzqmf add new parameters of existing stack, will we break updating existing pipeline created by old version?

Reply:
The newly parameters will not affect the old version pipeline update.